packages/eas-cli/src/user/User.ts

```diff
@@ -13,6 +13,8 @@ export function getActorDisplayName(user?: Actor): string {
       return user.username;
     case 'Robot':
       return user.firstName ? `${user.firstName} (robot)` : 'robot';
+    case 'SSOUser':
+      return 'sso';
```

Comment:
An SSOUser will have a username, so  user.username should be returned here

Reply:
Thanks!